queries/python/indents.scm

```diff
@@ -0,0 +1,14 @@
+[
+  (function_definition)
+  (if_statement)
+  (with_statement)
+  (try_statement)
+  (for_statement)
+  (for_in_clause)
+  (class_definition)
+] @indent
+
+[
+  "elif"
+  "else"
+] @branch
```

Comment:
could there be things like finally clause here ? or catch ?

Reply:
I have added finally and "except", since python doesn't have catch